paasta_tools/utils.py

```diff
@@ -659,6 +715,40 @@ def get_fsm_deploy_pipeline(self):
             raise PaastaNotConfiguredError(
                 'Could not find fsm_deploy_pipeline in configuration directory: %s' % self.directory)
 
+    def get_log_writer(self):
+        """Get the log_writer configuration out of global paasta config
+
+        :returns: The log_writer dictionary.
+        """
+        try:
+            return self['log_writer']
+        except KeyError:
+            raise PaastaNotConfiguredError('Could not find log_writer in configuration directory: %s' % self.directory)
```

Comment:
If you don't patch things properly, the tests would read your dev box's /etc/paasta, which is undesirable.

You'd at least need to patch `load_system_paasta_config` or `os.environ['PAASTA_SYSTEM_CONFIG_DIR']` to avoid that, and if you're going to patch out something, you might as well be explicit about its value in the test.